vector/src/main/java/im/vector/fragments/VectorRoomDetailsMembersFragment.java

```diff
@@ -926,8 +944,21 @@ public void onNetworkError(Exception e) {
                                     }
 
                                     @Override
-                                    public void onMatrixError(MatrixError e) {
-                                        onError(e.getLocalizedMessage());
+                                    public void onMatrixError(final MatrixError e) {
+                                        if (null != getRiotActivity()) {
+                                            getRiotActivity().runOnUiThread(new Runnable() {
+                                                @Override
+                                                public void run() {
+                                                    if (getRiotActivity() != null && MatrixError.M_CONSENT_NOT_GIVEN.equals(e.errcode)) {
```

Comment:
@bmarty Why don't you check "if (MatrixError.M_CONSENT_NOT_GIVEN.equals(e.errcode))" before calling `runOnUiThread`?

Reply:
Yes, it is possible